autometa/common/exceptions.py

```diff
@@ -150,7 +113,16 @@ def __str__(self):
         str
             Message written alongside raising the exception
         """
-        return self.value
+        message = """
+
+        NCBI databases nodes.dmp, names.dmp and merged.dmp are out of sync with each other.
+
+        To download the databses visit: ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/taxdump.tar.gz
```

Comment:
The url you have provided is a path to a tarball not a page to visit

```suggestion
        Up-to-date databases may be downloaded at ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/taxdump.tar.gz
```